extensions/amp-lightbox-gallery/0.1/test/test-amp-lightbox-gallery.js

```diff
@@ -1,37 +1,69 @@
-import {installLightboxGallery} from '../amp-lightbox-gallery';
+import {htmlFor} from '#core/dom/static-template';
 
-const TAG = 'amp-lightbox-gallery';
+// import {installLightboxGallery} from '../amp-lightbox-gallery';
```

Comment:
Perhaps not quite ready?

Reply:
That was a WIP to try and get unit tests going. After working with Alan for a bit we decided that it requires too much mocking so isn't worth it for write the unit tests. The e2e tests do cover this functionality.